examples/advanced/example.js

```diff
@@ -284,6 +288,28 @@ async function initMap(center) {
     $hoursExpanded.slideToggle(200);
     $toggleButton.html(isExpanded ? "▼" : "▲");
   });
+
+  // Handle travel mode selection
+  $(".travel-mode-option").click(function () {
+    // Remove active class from all options
+    $(".travel-mode-option").removeClass("active");
+
+    // Add active class to the clicked option
+    $(this).addClass("active");
+
+    // Get the selected travel mode
+    const selectedMode = $(this).data("mode");
+
+    // Update the current travel mode
+    if (selectedMode === "DRIVING") {
+      currentTravelMode = travelMode.DRIVING;
+    } else if (selectedMode === "WALKING") {
+      currentTravelMode = travelMode.WALKING;
+    }
```

Comment:
Nit: The API doesn't do a type check against that enum, so you could just use the `selectedMode` string directly